### ISSUE TITLE
Linux

### ISSUE THREAD

**Issue by Redo11** [mcedit/mcedit2#519]
i downloaded Mac Os in hope i will find standard unix binary. Sadly, no, but WHY YOU DON'T SUPPORT LINUX? Reasons why im angry?:
1. you use libraries COMMONLY used in linux
2. You already ha UNIX LIKE version, easier to port
3. Linux is starting to be on popular list. A lot of devs started developing for it. You know the stuff, i could write 5 paraghraps.

And the last one from guidelines, what i just ignored while writing it. Why are you talking about LINUX as a EXISTANT version, while IT IS NOT availbe to download from official website?!
Thats enouth anger about linux for me today. Back to wine...

**Comment by Cardtastic** [comment]
I agree with you that they should support Linux, also because a lot of the code is python code and they would only have to change a few things, but I don't think that's a reason to get angry. Getting angry rarely makes people do something, especially on the internet. If you had just calmly made your point clear, they might have already answered you.
